addon/mixins/sortable-item.js

```diff
@@ -308,14 +308,18 @@ export default Mixin.create({
     const groupDirection = this.get('group.direction');
     let dragOrigin;
     let elementOrigin;
+    let scrollOrigin;
 
     if (groupDirection === 'x') {
       dragOrigin = getX(startEvent);
       elementOrigin = this.get('x');
+      scrollOrigin = $(this.element).offset().left;
 
       return event => {
         let dx = getX(event) - dragOrigin;
-        let x = elementOrigin + dx;
+        let elementX = this.get('x') - this.element.offsetLeft;
+        let scrollX = $(this.element).offset().left - elementX;
+        let x = elementOrigin + dx + (scrollOrigin - scrollX);
```

Comment:
The `x` computed property uses scroll offset (line 120). Can the usage be unified? Should it be applied to the `y` computed property also?